src/main/java/gov/usgs/traveltime/TTService.java

```diff
@@ -0,0 +1,274 @@
+package gov.usgs.traveltime;
+
+import gov.usgs.processingformats.TravelTimeData;
+import gov.usgs.processingformats.TravelTimeException;
+import gov.usgs.processingformats.TravelTimePlotDataBranch;
+import gov.usgs.processingformats.TravelTimePlotDataSample;
+import gov.usgs.processingformats.TravelTimePlotRequest;
+import gov.usgs.processingformats.TravelTimeReceiver;
+import gov.usgs.processingformats.TravelTimeRequest;
+import gov.usgs.processingformats.TravelTimeService;
+import gov.usgs.traveltime.tables.TauIntegralException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.NavigableMap;
+
+public class TTService implements TravelTimeService {
+  /** A String containing the earth model path for the locator, null to use default. */
+  private String modelPath = null;
+
+  /** A String containing the serialized path for the locator, null to use default. */
+  private String serializedPath = null;
+
+  /**
+   * The TTService constructor. Sets up the earth model path.
+   *
+   * @param modelPath A String containing the earth model path to use
+   */
+  public TTService(String modelPath, String serializedPath) {
+    this.modelPath = modelPath;
+    this.serializedPath = serializedPath;
+  }
+
+  /**
+   * Function to get a travel time using the provided input, implementing the TravelTime service
+   * interface.
+   *
+   * @param request a Final TravelTimeRequest containing the TravelTime request
```

Comment:
This is totally minor, but I really have no idea of what you mean by "Final" or what the difference is betweem TravelTimeRequest and TravelTime request, just saying this comment is like saying the blue sky is blue...how is that for snarky?! ;-) happy friday!

Reply:
Final, as in `final TravelTimeRequest request`, the parameter is marked as final when it's passed in

Reply:
still doesn't make sense